dynamodb-stream-reader/src/test/java/no/unit/nva/elasticsearch/ConstantsTest.java

```diff
@@ -0,0 +1,15 @@
+package no.unit.nva.elasticsearch;
```

Comment:
The test adds no real benefit as it is covered by other tests

Reply:
The test is silly, but it raised codecoverage from 0.97 to 1.00 - replaced by annotation...